sdk/ai/azure-ai-inference/azure/ai/inference/models/_models.py

```diff
@@ -194,6 +194,8 @@ class ChatCompletionsNamedToolChoice(_model_base.Model):
     """A tool selection of a specific, named function tool that will limit chat completions to using
     the named function.
 
+    Readonly variables are only populated by the server, and will be ignored when sending a request.
+
     :ivar type: The type of the tool. Currently, only ``function`` is supported. Required. Default
```

Comment:
I hate these auto-inserted sentences ("Readonly variables are only populated by the server, and will be ignored when sending a request."). I remove them manually after every re-emit of the SDK. I tried to convince the Azure SDK Python team to remove it. They remove it for a while, then it came back. Not sure exactly why.